Dockerfile

```diff
@@ -19,166 +20,291 @@
+COPY docker/nodejs.sh /etc/ort/bash_modules
+
+#------------------------------------------------------------------------
+# GOLANG - BUild as a separate component
+FROM build AS gobuild
+
+ARG GO_DEP_VERSION=0.5.4
+ARG GO_VERSION=1.17.3
+ENV GOPATH=/opt/go
+RUN curl -L https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /opt -xz
+ENV PATH=/opt/go/bin:$PATH
+RUN go version
+RUN curl -ksS https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | bash
+RUN echo "add_local_path /opt/go/bin:$PATH" > /etc/ort/bash_modules/go.sh
+
+#------------------------------------------------------------------------
+# HASKELL STACK
+FROM build AS haskellbuild
+
+ARG HASKELL_STACK_VERSION=2.1.3
+RUN curl -ksS https://raw.githubusercontent.com/commercialhaskell/stack/v$HASKELL_STACK_VERSION/etc/scripts/get-stack.sh | bash -s -- -d /usr/bin
+
+#------------------------------------------------------------------------
+# REPO / ANDROID SDK
+FROM build AS androidbuild
+
+ARG ANDROID_SDK_VERSION=6858069
+ENV ANDROID_HOME=/opt/android-sdk
+
+RUN curl -ksS https://storage.googleapis.com/git-repo-downloads/repo > /usr/bin/repo \
+    && chmod a+x /usr/bin/repo
+
+RUN curl -Os https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip \
+    && unzip -q commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip -d $ANDROID_HOME \
+    && rm commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip \
+    && PROXY_HOST_AND_PORT=${https_proxy#*://} \
+    && if [ -n "$PROXY_HOST_AND_PORT" ]; then \
+       # While sdkmanager uses HTTPS by default, the proxy type is still called "http".
+       SDK_MANAGER_PROXY_OPTIONS="--proxy=http --proxy_host=${PROXY_HOST_AND_PORT%:*} --proxy_port=${PROXY_HOST_AND_PORT##*:}"; \
+    fi \
+    && yes | $ANDROID_HOME/cmdline-tools/bin/sdkmanager $SDK_MANAGER_PROXY_OPTIONS \
+       --sdk_root=$ANDROID_HOME "platform-tools" "cmdline-tools;latest"
+COPY docker/android.sh /etc/ort/bash_modules
+
+#------------------------------------------------------------------------
+# SCANCODE - Build scancode as a separate component
+FROM pythonbuild AS scancodebuild
+
+ARG SCANCODE_VERSION="3.2.1rc2"
+ENV SCANCODE_URL "https://github.com/nexB/scancode-toolkit/archive/refs/tags/"
+ENV PATH=/opt/python/shims:$PATH
+
+RUN mkdir -p /opt/scancode \
+    && curl -ksSL ${SCANCODE_URL}/v${SCANCODE_VERSION}.tar.gz | tar -C /opt/scancode -xz --strip-components=1 \
+    && cd /opt/scancode \
+    && PYTHON_EXE=python3 ./configure \
+    # cleanup unneeded installed binaries
+    && rm -rf /opt/scancode/thirdparty
+
+#------------------------------------------------------------------------
+# Main container
+FROM eclipse-temurin:11-jre
+
+# ORT
+COPY --from=ortbuild /opt/ort /opt/ort
+COPY docker/bash_bootstrap.sh /etc/ort/bash_bootstrap.sh
+COPY docker/ort-wrapper.sh /usr/bin/ort
+RUN chmod 755 /usr/bin/ort
+
+# Python
+COPY --from=pythonbuild /opt/python /opt/python
```

Comment:
Good question. 
pyenv take care of static python building, but i'm not sure about missing shared libraries.
Checking...

Reply:
Ok, now i got a confident answer.

With latest commit addition ( netbase ), here's the results after running Python tests ( Read to the end ):

```
== Tests result: FAILURE ==

379 tests OK.

1 test failed:
test_ssl

25 tests skipped:
test_curses test_dbm_ndbm test_devpoll test_gdb test_idle
test_kqueue test_msilib test_ossaudiodev test_smtpnet
test_socketserver test_startfile test_tcl test_timeout test_tix
test_tk test_ttk_guionly test_ttk_textonly test_turtle
test_urllib2net test_urllibnet test_winconsoleio test_winreg
test_winsound test_xmlrpc_net test_zipfile64

2 tests run no tests:
test_dtrace test_future4

```
The test_ssl fails is a upstream bug on the test itself:
https://bugs.python.org/issue40018

Since Ubuntu Focal is using openssl 1.1.1e, this test failed correctly following bug description.
So i can say we're good